rules/SAML-configuration-mapping.js

```diff
@@ -0,0 +1,33 @@
+function (user, context, callback) {
+  const CLIENTS = {
+    'wgh8S9GaE7sJ4i0QrAzeMxFXgWZYtB0l': 'sage-intacct',  // Sage Intacct
+    'R4djNlyXSl3i8N2KXWkfylghDa9kFQ84': 'thinksmart',    // mozilla.tap.thinksmart.com
+  }
+
+  const client = CLIENTS[context.clientID];
+
+  switch(client) {
+    case undefined:
+      break;
+    case 'sage-intacct':
+      user.company_name = 'MOZ Corp';
+
+      context.samlConfiguration.mappings = {
```

Comment:
We should probable add this in case the object is missing from `context`
```suggestion
      context.samlConfiguration = context.samlConfiguration || {};
      context.samlConfiguration.mappings = {
```

Reply:
We can, although none of the other rules do this. I'm pretty sure it sets `context.samlConfiguration` automatically.  :)

Reply:
Okay, made another push that cleans things up a bit and has a protective measure in there for you.  :)